itest/rust/src/builtin_tests/string/gstring_test.rs

```diff
@@ -69,12 +69,14 @@ fn string_chars() {
     // Empty tests regression from #228: Null pointer passed to slice::from_raw_parts().
     let string = GString::new();
     assert_eq!(string.chars(), &[]);
+    assert_eq!(string, GString::from(&[][..]));
 
     let string = String::from("some_string");
     let string_chars: Vec<char> = string.chars().collect();
     let gstring = GString::from(string);
 
     assert_eq!(string_chars, gstring.chars().to_vec());
+    assert_eq!(gstring, GString::from(&string_chars[..]));
```

Comment:
Also here, a separate variable with an expressive name. The fact that you operate on `&[char]` slices here is very well-hidden, it needn't be.